dev/integration_tests/ios_host_app/Host/Base.lproj/LaunchScreen.storyboard

```diff
@@ -1,7 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13122.16" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="01J-lp-oVM">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This was moved from [dev/integration_tests/ios_add2app/ios_add2app/Launch Screen.storyboard](https://github.com/flutter/flutter/pull/61007/files#diff-95f972d78b529be67ee75548a2086d30) but git didn't follow the rename.